WordPress/Classes/Utility/WPTableViewHandler.m

```diff
@@ -151,15 +151,18 @@ - (void)refreshTableViewPreservingOffset
         }
         newIndexPath = [self.resultsController indexPathForObject:obj];
 
-        CGRect originalCellFrame = [[visibleCellFrames objectAtIndex:i] CGRectValue];
-        if (newIndexPath) {
-            // Since we still have one of the originally visible objects,
-            // preserve the original cell frame's origin, relative to the original content offset.
-            originalCellOriginOffsetDelta += originalOffset.y - originalCellFrame.origin.y;
-            break;
-        } else {
-            originalCellOriginOffsetDelta -= originalCellFrame.size.height;
+        if (visibleCellFrames.count >= i) {
+            CGRect originalCellFrame = [[visibleCellFrames objectAtIndex:i] CGRectValue];
+            if (newIndexPath) {
+                // Since we still have one of the originally visible objects,
+                // preserve the original cell frame's origin, relative to the original content offset.
+                originalCellOriginOffsetDelta += originalOffset.y - originalCellFrame.origin.y;
+                break;
+            } else {
+                originalCellOriginOffsetDelta -= originalCellFrame.size.height;
+            }
         }
+
```

Comment:
Doing this would solve the if/else issue no?
```suggestion
            if (visibleCellFrames.count < i) {
                break;
            }
            
            CGRect originalCellFrame = [[visibleCellFrames objectAtIndex:i] CGRectValue];
            if (newIndexPath) {
                // Since we still have one of the originally visible objects,
                // preserve the original cell frame's origin, relative to the original content offset.
                originalCellOriginOffsetDelta += originalOffset.y - originalCellFrame.origin.y;
                break;
            } else {
                originalCellOriginOffsetDelta -= originalCellFrame.size.height;
            }
```